docs/data/toolpad/building-ui/custom-components.md

```diff
@@ -6,15 +6,15 @@
 
 In order to create a new custom component locate **+** button in the Explorer near Components section and click it:
 
-<img src="/static/toolpad/docs/custom-component-1.png" width="241px" alt="Custom component" />
+<img src="/static/toolpad/docs/building-ui/ui-10.png" width="253px" alt="Custom component" />
 
 Choose the name for your component and click **Create**:
 
-<img src="/static/toolpad/docs/custom-component-2.png" width="460px" alt="Custom create" />
+<img src="/static/toolpad/docs/building-ui/ui-11.png" width="434px" alt="Custom create" />
 
 You can now use code editor (on the left) and live preview (on the right) to define your custom component:
 
-<img src="/static/toolpad/docs/custom-component-3.png" width="990px" alt="Custom component IDE" />
+<img src="/static/toolpad/docs/building-ui/ui-12.png" width="1253px" alt="Custom component IDE" />
```

Comment:
🤔 The Monaco editor in this screenshot has red squigly lines under the imports. I wonder what's going on, can't seem to reproduce in my local setup. Did you wait long enough for the typings to load?

Reply:
I am able to reproduce it here: https://master--toolpad.mui.com/_toolpad/app/cl9pj1woq0001l59l5y3kztln/codeComponents/vh03lk8 - after reload it's red, but once you focus text it goes away 🤔 There are no errors in console so I'm not sure what's going on either.

Anyways, thanks for spotting, I will retake clean screen now

Reply:
Hmm, I can't reproduce on master neither